entities/sendlog.go

```diff
@@ -1,24 +1,25 @@
 package entities
 
-import "time"
+import (
+	"time"
+)
 
 const (
 	// SendLogStatusFailed status used when sender consumer fails to send email
-	SendLogStatusFailed     = "failed"
+	SendLogStatusFailed = "failed"
 	// SendLogStatusSuccessful status used when sender consumer succeeded sending the mail
 	SendLogStatusSuccessful = "successful"
 
 	// SendLogDescriptionOnSuccessful description used when sender succeeded sending the mail
-	SendLogDescriptionOnSuccessful     = "Email sent successfully"
+	SendLogDescriptionOnSuccessful = "Email sent successfully"
 	// SendLogDescriptionOnSesClientError description used when sender failed to create ses client
 	SendLogDescriptionOnSesClientError = "Unable to create ses client"
 	// SendLogDescriptionOnSendEmailError description used when ses client fails to send the email
 	SendLogDescriptionOnSendEmailError = "Unable to send email"
 )
 
 type SendLog struct {
-	ID           int64     `json:"id" gorm:"column:id; primary_key:yes"`
-	UUID         string    `json:"uuid" gorm:"column:uuid; index"`
+	UID          string    `json:"uid" gorm:"column:uid; primary_key:yes"`
```

Comment:
We can name this `ID`. Also instead of `string` type, will it work if we put it to be ksuid.KSUID type?

Reply:
Yes i think we can put this param to be KSUID type, it's working with db driver. @gudgl @FilipNikolovski

Reply:
Yesterday i tried but it didnt work.. now it works